apps/src/gamelab/animationListModule.js

```diff
@@ -691,24 +694,37 @@ function loadPendingFramesFromSource(key, props, callback) {
  * Given a key/serialized-props pair for an animation, work out where to get
  * the spritesheet.
  * @param {!AnimationKey} key
- * @param {!SerializedAnimationProps} props
+ * @param {object} animationList - Used to access props and check if src url
+ *        exists in this project and is deleted
```

Comment:
Might want to drop this comment about checking if it's deleted - I don't think we do this anymore?